tests/Jenkins/dirac_ci.sh

```diff
@@ -133,8 +133,25 @@ installSite() {
   cd -
 
 
-  echo "==> Done installing, now configuring"
+  echo "==> Sourcing bashrc"
   source "${SERVERINSTALLDIR}/bashrc"
+
+  echo "==> Installing main branch of diracx"
+  for wheel_name in core client cli; do
+    if [[ -n "${DIRACX_CUSTOM_SOURCE_PREFIXES:-}" ]]; then
+      wheels=( $(find "${DIRACX_CUSTOM_SOURCE_PREFIXES}" -name "diracx_${wheel_name}-*.whl") )
+      if [[ ! ${#wheels[@]} -eq 1 ]]; then
+          echo "ERROR: Multiple wheels found for ${package_name} in ${dir}"
+          exit 1
+      fi
+      pip install "${wheels[0]}"
+    else
+      pip install "git+https://github.com/DIRACGrid/diracx.git@main#egg=diracx-${wheel_name}&subdirectory=diracx-${wheel_name}"
```

Comment:
Why not simply

```suggestion
      pip install diracx-core
      pip install diracx-client
```
?

Reply:
because we want to install from the `main` branch of the repository